Makefile

```diff
@@ -0,0 +1,13 @@
+all: build
+
+build:
+	@docker build --tag=joeybaker/syncthing:latest .
+
+base:
+	@docker pull ubuntu:14.04
+
+rebuild: base
+	@docker build --tag=joeybaker/syncthing:latest .
+
+release: rebuild
+	@docker build --tag=joeybaker/syncthing:0.12.11 .
```

Comment:
hmmm… I think so… https://docs.docker.com/engine/reference/builder/#arg should do the trick? The only problem with the makefile approach though is that the repo can't auto-deploy to docker hub on push to master. What do you think?